docs/UserGuide.md

```diff
@@ -132,18 +132,26 @@ Options:
 -dM dosageMorning       Morning dosage of medication
 -dA dosageAfternoon     Afternoon dosage of medication
 -dE dosageEvening       Evening dosage of medication
--r remarks              Additional remarks on medication
 -rep                    How often to take medication (eg: Supply a number from 1 to 7)
+-r remarks              Additional remarks on medication
+-h                      Prints this help message
 ```
 
-> Tip: The dosage and remarks tags are optional.
 
-> Note: Remarks will default to Nil if it is not specified.
+<div class="tip-box">
+:bulb: <strong>Tip: </strong>
+The dosage and remarks tags are optional.
+</div>
+
+<div class="note-box">
+:notebook: <strong>Note: </strong>
+Remarks will default to Nil if it is not specified.
+</div>
 
 Examples:
 
-* `add -n Medication A -q 5000 -e 01/07/25 -dM 500 -dA 250 -r cause_dizziness -rep 1`
-* `add -n Medication B -q 1000 -e 30/09/24 -dM 500 -dA 250 -dE 50 -rep 6`
+* `meditracker> add -n Medication A -q 5000 -e 01/07/25 -dM 500 -dA 250 -r cause_dizziness -rep 1`
```

Comment:
In this line of example, might consider adding -dE too since this argument has now become compulsory.

Reply:
Yes, good catch. Included -dE in  aff146c